locales/en.json

```diff
@@ -291,295 +291,295 @@
     "log_user_group_add": "Add '{}' group",
     "log_user_group_delete": "Delete '{}' group",
     "log_user_group_update": "Update '{}' group",
-    "log_user_update": "Update information of '{}' user",
+    "log_user_update": "Update info of '{}' user",
```

Comment:
I would keep "information" ...

Reply:
"info" should be in consistent use, it saves on space similar to "admin" over administrator.